web/src/OverviewPane.tsx

```diff
@@ -40,31 +40,40 @@ let ServicesContainer = styled.div`
   overflow: auto;
 `
 
-function PinnedServicesDivider() {
-  return (
-    <ServicesDividerRoot>
-      <GridDividerPinSvg style={{ marginLeft: "28px" }} />
-      <ServicesLabel>Pinned Resources</ServicesLabel>
-    </ServicesDividerRoot>
-  )
+function PinnedResources(items: OverviewItem[]) {
+  return items?.length ? (
+    <div className={"resources-container pinned"}>
```

Comment:
Let's get rid of all the CSS class names in this PR. None of them are used for styling. Once this is a React.Fragment, you should be able to use the react component itself as a selector.